chromadb/api/segment.py

```diff
@@ -882,7 +883,7 @@ def _records(
 
         record = t.OperationRecord(
             id=id,
-            embedding=embeddings[i] if embeddings else None,
+            embedding=embeddings[i] if embeddings is not None else None,
```

Comment:
not specific to this change but if embeddings returns false for 0-dim embeddings which could break here

Reply:
To clarify, you're not suggesting any specific change for this PR, right?